### PR TITLE
Add service error pages

### PR DESCRIPTION
**Pull request by simonwhatley**
This PR adds error pages for:

- [Page not found (404)](https://itt-mentoring-beta-pr-74.herokuapp.com/404) - sign in first to view
- [Unexpected error (500)](https://itt-mentoring-beta-pr-74.herokuapp.com/500)
- [Service unavailable (503)](https://itt-mentoring-beta-pr-74.herokuapp.com/503)
- [User unauthorised](https://itt-mentoring-beta-pr-74.herokuapp.com/unauthorised)

### CODE REVIEW

**Review by petejobeskinandcarta — CHANGES_REQUESTED** [review]
@LeaWelch has done copy for the unathorised page, it's on the ticket here.

https://trello.com/c/3hYeqkeQ/337-account-is-not-authorised-page

**Review by LeaWelch — DISMISSED** [review]

File: app/views/errors/503.njk [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends "layouts/error.njk" %}
+
+{% set title = "Sorry, the service is unavailable" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ title }}</h1>
+      <p class="govuk-body">You’ll be able to use the service from 3pm on Tuesday, 16 April 2024.</p>
```
[the remark this answers is not in the document]
Comment (simonwhatley) [reply app/views/errors/503.njk]: @LeaWelch, I've removed the contractions as requested.

**Comment by simonwhatley** [comment]
@LeaWelch @petejobeskinandcarta this is ready for review

File: app/views/errors/503.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (olioldfield) [reply app/views/errors/503.njk]: Does this mean the SLA needs to be updated to remove the contraction too?

You’ll get a response within 5 working days.

[the remark this answers is not in the document]
Comment (simonwhatley) [reply app/views/errors/503.njk]: My understanding is that negative contractions are a no-no. Others are okay.

[the remark this answers is not in the document]
Comment (simonwhatley) [reply app/views/errors/503.njk]: We should be consistent in the service, so we will need to change the footer content.

**Comment by simonwhatley** [comment]
[Account is not authorised page](https://trello.com/c/3hYeqkeQ/337-account-is-not-authorised-page)

**Review by olioldfield — COMMENTED** [review]
All looks good, but one comment on the account unauthorised page:

If you work with us and think your account should be authorised, email [ittmentor.funding@education.gov.uk](mailto:ittmentor.funding@education.gov.uk).

I'm not sure if "if you work with us" is quite right - maybe:

If you think your account should be authorised, email
[ittmentor.funding@education.gov.uk](mailto:ittmentor.funding@education.gov.uk).

**Pull request merged**